internal/api/post_test_plan_metadata.go

```diff
@@ -12,9 +12,9 @@ type Timeline struct {
 }
 
 type TestPlanMetadataParams struct {
-	Version     string            `json:"version"`
-	SplitterEnv map[string]string `json:"splitter_env"`
-	Timeline    []Timeline        `json:"timeline"`
+	Version  string            `json:"version"`
+	Env      map[string]string `json:"env"`
```

Comment:
I think we can just call it `env` instead of `bktec_env`?

Reply:
yeah i had it as `bktec_env` as the prev name was `splitter_env`.  Happy for just `env`